src/main/kotlin/no/nav/familie/ef/sak/tilbakekreving/TilbakekrevingController.kt

```diff
@@ -54,7 +54,7 @@ class TilbakekrevingController(
         return Ressurs.success(tilbakekrevingService.finnesÅpenTilbakekrevingsBehandling(behandlingId))
     }
 
-    @GetMapping("/{behandlingId}/finnesFlereTilbakekrevingerValgtSisteÅr")
+    @GetMapping("/{behandlingId}/finnes-flere-tilbakekrevinger-valgt-siste-aar")
```

Comment:
Blir det trøbbel i frontend? 
Legge til ny aa først, så  bytte, så slette?

Reply:
Men hvis den ikke fungerer i det hele tatt er det kanskje ikke så farlig...

Reply:
Det er allerede trøbbel i frontend 😅 Men de merker det ikke for det er togglet av